### DISCUSSION TITLE
Edge sees 8.2 Update, but Firefox doesnt

### DISCUSSION THREAD

**Discussion by Senajs** [aclap-dev/video-downloadhelper#67] [Help and Q&A]
### Your question

This morning my Edge got the 8.2 Update - but no matter what I try firefox still sees only 8.1 - any idea why ?

**Comment by GrampaWildWilly** [comment]
Because it hasn't been released yet?  Or more precisely, that Mozilla hasn't gotten around to approving it for release on their add-on repository.  Patience, Grasshopper.

**Reply by Senajs** [reply]
thanks man - i assumed that such an announcement would come after the platforms had published it :-)
